YaLibs/YaToolsIDALib/Repository.cpp

```diff
@@ -750,20 +770,34 @@ bool Repository::rebase(const std::string& upstream, const std::string& destinat
 
 bool Repository::add_file_to_index(const std::string& path)
 {
+    fs::path p = path;
+    if (!include_idb_ &&
+        (path == get_current_idb_name() ||
+        path == get_original_idb_name()))
+    {
+        p = ".gitignore";
+        f.open(p, std::fstream::out);
```

Comment:
missing std::fstream f

Reply:
I'm sorry, I thought I built the code after this but apparently I didn't.  Fixed in 8000c16d257bb85046304531e5aac8ea1c3fe6f4